### PR TITLE
feat(yarn): add options

### LINKED ISSUE #1611: [YARN] Missing Specs
### Sanity checks

- [X] My issue relates to a specific CLI completion spec (e.g. `git checkout` is missing options in `git` completion spec). If your issue is more general, please create your issue here: [withfig/fig](https://github.com/withfig/fig/issues/new?assignees=&labels=NEED_TO_LABEL&template=main_issue_template.yml)
- [X] I have searched [github.com/withfig/autocomplete/issues](https://github.com/withfig/autocomplete/issues?q=) and there are no duplicates of my issue
- [X] I would like to work on this.

### What CLI tool does this relate to?

yarn

### Which statement makes the most sense?

This completion spec isn't complete

### Adding subcommands for policies in yarn
Currently there's no yarn's subcommand for policies. Here are the missing command which I've already added into 
**yarn.ts**
- **yarn policies set-version**
- **yarn policies set-version --rc**

### Adding subcommands for team in yarn
Here are the **missing** commands for team:
- yarn team create <scope:team>
- yarn team destroy <scope:team>
- yarn team add <scope:team> <user>
- yarn team remove <scope:team> <user>
- yarn team list <scope>|<scope:team>

**I've already created a PR targeting this issues, please take a look and assign me to this :)**

### PR DESCRIPTION
**Pull request by vatana7**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Adding more specs to yarn

**Important**
This is a code solution submitted  for Closes #1611

### CODE REVIEW

File: src/yarn.ts [hunk 1]
```diff
@@ -1148,6 +1148,16 @@ const completionSpec: Fig.Spec = {
...
       description: "Defines project-wide policies for your project",
+      subcommands: [
+        {
+          name: "set-version",
+          description: "Will download the latest stable release",
+        },
+        {
+          name: "set-version --rc",
+          description: "Will download the latest rc release",
+        },
```
Comment (fedeci) [inline src/yarn.ts:1159]:
```suggestion
      options: [{
        name: "--rc",
        description: "Download the latest rc release",
      }],
        },
```

File: src/yarn.ts [hunk 2]
```diff
@@ -1254,6 +1264,49 @@ const completionSpec: Fig.Spec = {
     {
       name: "team",
       description: "Maintain team memberships",
+      subcommands: [
+        {
+          name: "create",
+          description: "Create a new team",
+          args: {
+            name: "<scope:team>",
+            isOptional: true,
```
Comment (fedeci) [inline src/yarn.ts:1273]:
```suggestion
```

File: src/yarn.ts [hunk 3]
```diff
@@ -1254,6 +1264,49 @@ const completionSpec: Fig.Spec = {
...
+          },
+        },
+        {
+          name: "destroy",
+          description: "Destroys an existing team",
+          args: {
+            name: "<scope:team>",
+            isOptional: true,
```
Comment (fedeci) [inline src/yarn.ts:1281]:
```suggestion
```

File: src/yarn.ts [hunk 4]
```diff
@@ -1254,6 +1264,49 @@ const completionSpec: Fig.Spec = {
...
+          },
+        },
+        {
+          name: "list",
+          description:
+            "If performed on an organization name, will return a list of existing teams under that organization. If performed on a team, it will instead return a list of all users belonging to that particular team",
+          args: {
+            name: "<scope>|<scope:team>",
+            isOptional: true,
```
Comment (fedeci) [inline src/yarn.ts:1306]:
```suggestion
```

File: src/yarn.ts [hunk 5]
```diff
@@ -1254,6 +1264,49 @@ const completionSpec: Fig.Spec = {
...
+            name: "<scope:team> <user>",
+            isOptional: true,
+          },
+        },
+        {
+          name: "remove",
+          description: "Remove a user from a team they belong to",
+          args: {
+            name: "<scope:team> <user>",
+            isOptional: true,
```
Comment (fedeci) [inline src/yarn.ts:1297]:
```suggestion
```

File: src/yarn.ts [hunk 6]
```diff
@@ -1254,6 +1264,49 @@ const completionSpec: Fig.Spec = {
...
+          },
+        },
+        {
+          name: "add",
+          description: "Add a user to an existing team",
+          args: {
+            name: "<scope:team> <user>",
+            isOptional: true,
+          },
```
Comment (fedeci) [inline src/yarn.ts:1290]:
```suggestion
          args: [{
            name: "<scope:team>",
          }, {
            name: "<user>",
          }],
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for the PR! I left a few things to get fixed before merging this in.

**Comment by vatana7** [comment]
Thanks! Can you label this as hacktoberfest PR please 🤟🏻

**Comment by SeparateRecords** [comment]
@vatana7 PRs are automatically counted for hacktoberfest because the repo itself has the label 🙂

**Comment by vatana7** [comment]
Everything should be able to merge now. Thanks for the suggestion!

**Comment by SeparateRecords** [comment]
Just one thing to fix up first! Either prettier or eslint is unhappy. Easy fix: `npx eslint --fix src/yarn.ts; npx prettier -w src/yarn.ts`

**Comment by vatana7** [comment]
What version of node does this project use?

**Comment by fedeci** [comment]
That's not strictly relevant. I believe node@16 is good.

**Comment by SeparateRecords** [comment]
Any version of node should be fine for this. Personally I just use the latest (node@18) and only change per project if required

File: src/yarn.ts [hunk 7]
```diff
@@ -1254,6 +1266,49 @@ const completionSpec: Fig.Spec = {
...
+              name: "<user>",
+            },
+          ],
+        },
+        {
+          name: "remove",
+          description: "Remove a user from a team they belong to",
+          args: {
+            name: "<scope:team> <user>",
+          },
```
Comment (fedeci) [inline src/yarn.ts:1301]:
```suggestion
          args: [
            {
              name: "<scope:team>",
            },
            {
              name: "<user>",
            },
          ],
```

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**